### DISCUSSION TITLE
hlines/vlines index order

### DISCUSSION THREAD

**Discussion by cdacl** [lvjr/tabularray#52] [Q&A]
According to the documentation of tabularray package, we can use hlines={<index>}{<columns>}{<styles>} in mandatory argument to type more hlines/vlines. But I can't change the index order, for example:

```
\begin{tblr}{
		hlines = {2}{-}{red5},
		hlines = {1}{-}{blue5},
}
 Alpha & Beta & Gamma & Delta \\
 Epsilon & Zeta & Eta & Theta \\
 Iota & Kappa & Lambda & Mu \\
\end{tblr}
```
I got this:
![1](https://user-images.githubusercontent.com/87848848/127993697-ec71542a-5223-48b6-bb8c-bd5ad2cc5ecb.png)

The red line was missed.

**Comment by lvjr** [comment]
Yes, you must use index 1 first. I should mention this in the manual.

**Reply by cdacl** [reply]
OK. I got it. Thank you.

**Reply by lvjr** [reply]
Created issue #61.
